controllers/api/showRoutes.js

```diff
@@ -0,0 +1,60 @@
+const router=require('express').Router();
+const {Show} = require('../../models');
+
+// creates a new show
+
+router.post('/',async(req,res)=>{
+    try{
```

Comment:
I'm not as familiar with the try syntax, but this doesn't work as-is because you're not actually returning a response to the service, so when you try to create a show to add to the database, we'll sort of just be hung while waiting for a response.

FWIW, this code inside that route works (assuming you make the other changes):
```

Show.create({
      id: req.body.id,
      name: req.body.name,
      premiereDate:req.body.premiereDate,
      service_id:req.body.service_id
    })
    .then(dbShowData => res.json(dbShowData))
    .catch (err => {
        console.log(err);
        res.status(500).json(err);
    })

```